source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/CimDeserialization/ChargeLinkBundle/ChargeLinkCommandConverterTests.cs

```diff
@@ -114,16 +116,69 @@ public class ChargeLinkCommandConverterTests
             using var reader = XmlReader.Create(stream, new XmlReaderSettings { Async = true });
 
             // Act
-            var result = (ChargeLinksCommand)await sut.ConvertAsync(reader).ConfigureAwait(false);
+            var resultBundle = (ChargeLinksCommandBundle)await sut.ConvertAsync(reader).ConfigureAwait(false);
 
             // Assert
-            Assert.Equal("DocId_Valid_003", result.Document.Id);
-            Assert.Equal("rId_Valid_003", result.ChargeLinks.First().OperationId);
-            Assert.Null(result.ChargeLinks.First().EndDateTime);
+            var result = resultBundle.ChargeLinksCommands.First();
```

Comment:
nit: As we know there is exactly one charge link command in the test file:
```suggestion
            var result = resultBundle.ChargeLinksCommands.Single();
```